test/unit/test_encoder.py

```diff
@@ -24,6 +25,96 @@
 _DATA_LENGTH_ND = mx.nd.array([1, 2, 3, 4, 5, 6, 7, 8])
 
 
+def test_get_recurrent_encoder():
```

Comment:
Thanks for adding these tests! Could you add 1 test where we don't supply a conv_config (which is mostly the default)?